docs/release-announcement.md

```diff
@@ -0,0 +1,16 @@
+# Release announcement
+ 
+## Azure Industrial IoT Platform 2.7.20x 
```

Comment:
># Azure Industrial IoT Platform 2.7.20x  [](http://example.com/codeflow?start=1&length=40)

does the text still make sense?

Reply:
No, correct, this should change upon release of 2.8 and look forward to 2.9.  However, it would be a separate check in as the LTS text is still being worked on as part of another work item.